packages/libs/wdk-client/src/Components/AttributeFilter/FieldList.jsx

```diff
@@ -187,7 +187,7 @@ function FieldNode({ node, isActive, searchTerm, handleFieldSelect }) {
   }, [isActive, nodeRef.current, searchTerm]);
 
   return (
-    <Tooltip content={node.field.description} hideDelay={0}>
+    <Tooltip title={node.field.description} leaveDelay={0}>
```

Comment:
I forgot that this particular tooltip will render a small, empty yellow box when the title is null-ish. I wonder if we should open a follow-up ticket for the new `Tooltip` component to render null if its title prop is falsy?

Reply:
Done, let me know what you think of the solution!